Source/Core/MortonOrder.js

```diff
@@ -0,0 +1,85 @@
+import Check from "./Check.js";
+import defined from "./defined.js";
+import DeveloperError from "./DeveloperError.js";
+
+/**
+ * Morton Order (aka Z-Order Curve) helper functions.
+ * @see {@link https://en.wikipedia.org/wiki/Z-order_curve}
+ * @see {@link http://graphics.stanford.edu/~seander/bithacks.html#InterleaveBMN}
+ *
+ * @namespace MortonOrder
+ */
+var MortonOrder = {};
+
+/**
+ * @private
```

Comment:
I realize these are private methods, but would be nice to have a brief description of input and output.